### PR TITLE
Update bulk actions for trashed orders

### LINKED ISSUE #7937: 3.0 - Trashed orders cannot be permanently deleted with bulk actions
### Expected behavior
On the orders list table, I expect to have the same basic bulk actions available to me that I do in other list tables (such as posts).

### Actual behavior
When viewing "Trashed" orders, there is no way to delete all of them. Orders (applies to refunds as well):

image

Viewing "Trashed" downloads:

image

(I can Delete Permanently by selecting multiple items, or click the "Empty Bin/Trash" button and clean everything out in one go.

### Steps to reproduce the behavior
1) In EDD 3.0, delete some orders. This now moves them to a temporary "trash" location, added in 3.0.
2) Now go to the trashed orders (`/wp-admin/edit.php?post_type=download&page=edd-payment-history&status=trash`)
3) The trashed orders cannot be deleted in bulk (the action links are available, so individual orders can be deleted permanently, although the redirect takes me out of the trashed orders view).

### Information (if a specific version is affected):
PHP Version: 7.3.5
EDD Version (or branch): 3.0
WordPress Version: 5.4

### PR DESCRIPTION
**Pull request by robincornett**
Fixes #7937

Proposed Changes:
1. Changes the bulk actions list when viewing trashed orders.
2. Removes pagination if no orders match the current status.

### CODE REVIEW

File: includes/admin/payments/class-payments-table.php [hunk 1]
```diff
@@ -656,6 +656,7 @@ public function get_bulk_actions() {
 
 		if ( 'trash' === $this->get_status() ) {
 			$action['restore'] = __( 'Restore', 'easy-digital-downloads' );
+			$action['delete']  = __( 'Delete Permanently', 'easy-digital-downloads' );
 		} else {
 			$action['trash'] = __( 'Trash', 'easy-digital-downloads' );
```
Comment (spencerfinnell) [inline includes/admin/payments/class-payments-table.php:661]: Let's change this to "Move to Trash" to match core.

Comment (robincornett) [reply includes/admin/payments/class-payments-table.php:661]: @spencerfinnell  this is on the filtered view showing only items which are already in the trash. This is making the trashed orders table bulk actions equivalent to what is in Core (screenshots in the related issue).

Comment (spencerfinnell) [reply includes/admin/payments/class-payments-table.php:661]: @robincornett Sorry, the comment wasn't for the addition (line 659) but the unchanged line (now 661).

Core Posts:
Screen Shot 2020-05-20 at 11 51 50 AM

EDD Orders:
Screen Shot 2020-05-20 at 11 52 04 AM

Comment (robincornett) [reply includes/admin/payments/class-payments-table.php]: My apologies, I misunderstood. Yes, definitely a good call to change that as well. Thanks!

**Comment by spencerfinnell** [comment]
Failing test is unrelated:

```
1) EDD\Orders\Payment_Tests::test_get_payment_completed_date
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'2020-05-20 16:16'
+'2020-05-20 16:15'
```

**Pull request merged**